corsikaio/file.py

```diff
@@ -150,13 +165,16 @@ def close(self):
 
 class CorsikaCherenkovFile(CorsikaFile):
 
-    def __init__(self, path, mmcs=False):
-        super().__init__(path)
+    def __init__(self, path, mmcs=False, **kwargs):
+        super().__init__(path, **kwargs)
 
         self.EventClass = PhotonEvent
         self.mmcs = mmcs
 
     def parse_data_blocks(self, data_bytes):
+        if not self.parse_blocks:
```

Comment:
Do we need to check this? `parse_data_blocks` is never called if `parse_blocks` is false. Of course this could change at some point, or this is externally called, manually... Just wanted to make sure this is intentional.